### PR TITLE
webflow_percy

### PR DESCRIPTION
**Pull request by indulekha-deriv**
PR to enable percy for webflow

### CODE REVIEW

File: cypress/e2e/webflow/Percy/percy.cy.js [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+
+// Load URLs from the JSON file
+let urls
+if (Cypress.env('webflow_env') === 'staging') {
+  urls = require('./staging_urls.json')
+} else {
+  urls = require('./prod_urls.json')
+}
+function snapshot(url) {
+  cy.viewport(400, 1024)
```
Comment (munsei-deriv) [inline cypress/e2e/webflow/Percy/percy.cy.js:11]: is this standard size for mobile responsive? Based on which phone model resolution?

File: cypress/e2e/webflow/Percy/prod_urls.json [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+    "https://deriv.com/who-we-are/",
+    "https://deriv.com/responsible/",
+    "https://deriv.com/regulatory/",
+    "https://deriv.com/our-principles",
+    "https://deriv.com/why-choose-us",
+    "https://deriv.com/contact-us",
+    "https://deriv.com/help-centre/",
+    "https://deriv.com/payment-methods",
+    "https://deriv.com/careers/",
+    "https://deriv.com/terms-and-conditions/"
```
Comment (munsei-deriv) [inline cypress/e2e/webflow/Percy/prod_urls.json:13]: For the "deriv.com" url, perhaps we can use variable instead?

File: cypress/e2e/webflow/Percy/staging_urls.json [hunk 3]
```diff
@@ -0,0 +1,17 @@
...
+      "https://deriv-staged.webflow.io/who-we-are/",
+      "https://deriv-staged.webflow.io/responsible/",
+      "https://deriv-staged.webflow.io/regulatory-information",
+      "https://deriv-staged.webflow.io/our-principles",
+      "https://deriv-staged.webflow.io/why-choose-us",
+      "https://deriv-staged.webflow.io/contact-us",
+      "https://deriv-staged.webflow.io/help-center",
+      "https://deriv-staged.webflow.io/payment-methods",
+      "https://webflow.deriv.com/careers",
+      "https://deriv-staged.webflow.io/terms-and-conditions"
```
Comment (munsei-deriv) [inline cypress/e2e/webflow/Percy/staging_urls.json:13]: Same for this "deriv-staged.webflow.io" or "webflow.deriv.com", perhaps we can use variable instead?

File: cypress/e2e/webflow/Percy/percy.cy.js [hunk 4]
```diff
@@ -0,0 +1,36 @@
...
+describe('Visit URLs and Capture Percy Snapshots', () => {
+  urls.urls.forEach((url) => {
+    it(`Visits ${url} and captures Percy snapshot`, () => {
+      cy.log('WEBFLOW_ENV value:', Cypress.env('webflow_env'))
+      cy.visit(url)
+     // if (Cypress.env('webflow_env') === 'staging') {
+     //  cy.get('.contact-social_embed.is-whatsapp').should('be.visible', { timeout: 30000 })
+     // } else {
+     //   cy.c_waitForPageLoad()
+     // }
```
Comment (bilalmajid-deriv) [inline cypress/e2e/webflow/Percy/percy.cy.js:31]: If not required can we remove this.

File: cypress/e2e/webflow/Percy/percy.cy.js [hunk 5]
```diff
@@ -0,0 +1,36 @@
...
+      cy.wait(3000)
```
Comment (bilalmajid-deriv) [inline cypress/e2e/webflow/Percy/percy.cy.js:32]: can we wait for element visibility instead like the whatsapp icon?

File: cypress/e2e/webflow/Percy/percy.cy.js (hunk 1 above)
Comment (indulekha-deriv) [reply cypress/e2e/webflow/Percy/percy.cy.js]: Changed to standard viewports, Iphone-xr & macbook -16

File: cypress/e2e/webflow/Percy/prod_urls.json (hunk 2 above)
Comment (indulekha-deriv) [reply cypress/e2e/webflow/Percy/prod_urls.json]: Updated to use one single file with variable

File: cypress/e2e/webflow/Percy/percy.cy.js (hunk 5 above)
Comment (indulekha-deriv) [reply cypress/e2e/webflow/Percy/percy.cy.js:30]: There's on going issue in web flow where whatsApp icon is missing. Once it's fixed will update code to check for WhatsApp icon. https://deriv-group.slack.com/archives/C06J9KQN93J/p1713417903364539

File: cypress/e2e/webflow/Percy/percy.cy.js (hunk 4 above)
Comment (indulekha-deriv) [reply cypress/e2e/webflow/Percy/percy.cy.js]: removed

**Review by meeyun-deriv — APPROVED** [review]
LGTM

**Pull request merged**